apps/backend/src/index.ts

```diff
@@ -6,16 +6,21 @@ import authRoute from './router/auth';
 import dotenv from 'dotenv';
 import session from 'express-session';
 import passport from 'passport';
+import cookieParser from 'cookie-parser';
+
+export const COOKIE_MAX_AGE = 24 * 60 * 60 * 1000;
```

Comment:
maybe create a seperate file to store constats?

Reply:
moved to a separate file